shell/browser/api/electron_api_app.cc

```diff
@@ -1442,6 +1442,26 @@ void App::SetUserAgentFallback(const std::string& user_agent) {
   ElectronBrowserClient::Get()->SetUserAgent(user_agent);
 }
 
+#if defined(OS_WIN)
+
+bool App::IsRunningUnderARM64Translation() const {
+  USHORT processMachine = 0;
+  USHORT nativeMachine = 0;
+
+  auto IsWow64Process2 = reinterpret_cast<decltype(&::IsWow64Process2)>(
+      GetProcAddress(GetModuleHandle(L"kernel32.dll"), "IsWow64Process2"));
+
+  if (IsWow64Process2) {
```

Comment:
Should return `false` when `IsWow64Process2` is nullptr.

Reply:
It should already be like that, since if it's `nullptr`, `nativeMachine` will remain `0`, different from `IMAGE_FILE_MACHINE_ARM64`. But I'll add an explicit check. Thank you!